container_env_files/cgw.env

```diff
@@ -36,3 +36,9 @@ ALERTS_PROVIDER_SIGNING_KEY=''
 ALERTS_PROVIDER_API_KEY=''
 ALERTS_PROVIDER_ACCOUNT=''
 ALERTS_PROVIDER_PROJECT=''
+
+# Email handling
```

Comment:
Maybe you can specify the provider used, in case someone wants to pay/configure it?

Reply:
Makes sense, thanks! (done in[117a5ba](https://github.com/safe-global/safe-infrastructure/pull/126/commits/117a5baf5d5b083c55ca73e96bc6a24b5a5f9917))

Reply:
Thanks!